_shared/project/tox.ini

```diff
@@ -133,9 +137,9 @@ commands =
     lint: pydocstyle src tests bin
     lint: pycodestyle src tests bin
 {% endif %}
-    tests: coverage run -m pytest --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
+    tests: python -m pytest --cov --cov-append --cov-report= --cov-fail-under=0 {%- if cookiecutter.get("__parallel_unit_tests") %} --numprocesses logical {%- endif %} --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
```

Comment:
Updated unit tests command now that we're using `pytest-cov` and have optional support for running the unit tests in parallel